Typscript compile error - logger.d.ts missing in NPM package

### Version
3.5.0

### Reproduction link
[https://jsfiddle.net/chrisvfritz/50wL7mdz/](https://jsfiddle.net/chrisvfritz/50wL7mdz/)







### Steps to reproduce
Create default typescript project:

vue create test

? Please pick a preset: Manually select features
? Check the features needed for your project: Babel, TS, Router, Vuex, Linter
? Use class-style component syntax? No
? Use Babel alongside TypeScript (required for modern mode, auto-detected polyfills, transpiling JSX)? Yes
? Use history mode for router? (Requires proper server setup for index fallback in production) No
? Pick a linter / formatter config: Prettier
? Pick additional lint features: Lint on save
? Where do you prefer placing config for Babel, ESLint, etc.? In dedicated config files

cd test
npm run serve


### What is expected?
Expects to compile without errors.

### What is actually happening?
ERROR in D:/test/node_modules/vuex/types/index.d.ts(7,26):
7:26 Cannot find module './logger' or its corresponding type declarations.
     5 |
     6 | import { mapState, mapMutations, mapGetters, mapActions, createNamespacedHelpers } from "./helpers";
  >  7 | import createLogger from "./logger";
       |                          ^
     8 |
     9 | export * from "./helpers";
    10 | export * from "./logger";
ERROR in D:/test/node_modules/vuex/types/index.d.ts(10,15):
10:15 Cannot find module './logger' or its corresponding type declarations.
     8 |
     9 | export * from "./helpers";
  > 10 | export * from "./logger";

---
logger.d.ts file is missing in types folder when installed via npm.

<!-- generated by vue-issues. DO NOT REMOVE -->

--- a/package.json
+++ b/package.json
@@ -12,6 +12,7 @@
     "dist",
     "types/index.d.ts",
     "types/helpers.d.ts",
+    "types/logger.d.ts",
     "types/vue.d.ts"
   ],
   "scripts": {

